README.md

```diff
@@ -16,16 +23,178 @@ There have been, and are currently, several efforts in this area:
 
 This repository is an effort to bring together tool owners to standardize on a common specification for a description format.
 
-# Use Cases
+Please join the discussion in [Issues](https://github.com/webcomponents/custom-elements-json/issues).
 
-## Editor Support
+## Versioning
 
-Developers using custom elements should be able to get full-featured IDE support including auto-competion, hover-documentation, unknown symbol warnings, etc. These features should be available in HTML files, and in various template syntaxes via template-specific tools.
+The `custom-elements.json` follows a version number to allow for a gradual upgrade process.
 
-## Documentation
+- [Version 2](#version-2)
+- [Version 1](#version-1)
+
+### Version 2
+
+The main goal if this version is to address all key points of web-components.
+It shall serve as a starting point for adoption in tools and for web component authors.
+
+It is fully based on [Version 1](#version-1). 
+It adds all mentioned key points for web components.
+It on purpose does not define a type system. This is left to the next iteration.
+Therefore "type" should be seen as a type hint which can be presented to the end-user as documentation.
```

Comment:
I think we should put in some work to define the type field better

Reply:
added them via "Reference import" now 👍 
please take a look 🤗